corehq/form_processor/tests/test_gdpr_scrub_user_from_forms.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+from django.test import TestCase
+
+from corehq.blobs.tests.util import TemporaryFilesystemBlobDB
+from corehq.form_processor.interfaces.dbaccessors import FormAccessors
+from corehq.form_processor.tests.utils import use_sql_backend
+from corehq.apps.users.management.commands.gdpr_scrub_user_from_forms import Command
+from corehq.form_processor.models import XFormAttachmentSQL
+from corehq.form_processor.utils import TestFormMetadata
+from corehq.form_processor.utils import get_simple_wrapped_form
+import uuid
+
+DOMAIN = 'test-form-accessor'
+
+GDPR_SIMPLE_FORM = """<?xml version='1.0' ?>
+<data uiVersion="1" version="17" name="{form_name}" xmlns:jrm="http://dev.commcarehq.org/jr/xforms"
+    xmlns="{xmlns}">
+    <dalmation_count>yes</dalmation_count>
+    <n0:meta xmlns:n0="http://openrosa.org/jr/xforms">
+        <n0:deviceID>{device_id}</n0:deviceID>
+        <n0:timeStart>{time_start}</n0:timeStart>
+        <n0:timeEnd>{time_end}</n0:timeEnd>
+        <n0:username>{username}</n0:username>
+        <n0:userID>{user_id}</n0:userID>
+        <n1:appVersion xmlns:n1="http://commcarehq.org/xforms"></n1:appVersion>
+    </n0:meta>
+    {case_block}
+</data>"""
```

Comment:
Any reason not to just use the existing `SIMPLE_FORM` template?  Looks like the differences are `n0` instead of `n1`, and the `instanceID`, which both seems tangential to this test.